nwtopt.sh

```diff
@@ -19,16 +19,24 @@ source nwtenv/bin/activate
 echo
 conda-unpack
 python3 --version
+echo $PATH
+pwd
+echo
+ls nwtenv
+echo
+# echo -e ${PATH//:/\\n}
 echo ...Changed Python Environment...
 echo
 cp NWT_SUBMIT/NWTOPT_FILES/objective.py nwtenv/bin/objective.py
 cp run.sh nwtenv/bin/run.sh
 cp run.sh NWT_SUBMIT/PROJECT_FILES/run.sh
+mv NWT_SUBMIT/NWTOPT_FILES/HParams.py NWT_SUBMIT/PROJECT_FILES/HParams.py
 mv NWT_SUBMIT/NWTOPT_FILES/mfnwt NWT_SUBMIT/PROJECT_FILES/mfnwt
 mv NWT_SUBMIT/NWTOPT_FILES/nwtnum.txt NWT_SUBMIT/PROJECT_FILES/nwts/nwtnum.txt
 mv NWT_SUBMIT nwtenv/bin/NWT_SUBMIT
 echo ...Moved Necessary Files.....
 echo
-ls
 dir=$(pwd)
+export PYTHONPATH="${PYTHONPATH}:${dir}/nwtenv/bin/NWT_SUBMIT"
```

Comment:
FWIW I try not to rely on the `PYTHONPATH` var and, instead, smack my conda pack env at the front of the system path var `$PATH` like `export PATH=$PWD/nwtenv/bin:$PATH` but maybe you are relying on it for more than running the python binary?

Reply:
Yeah that was remnant code from trying to get the hyperparams to transfer to workers. I believe its the solution, but for this merge it is not necessary

Reply:
Fixed in most recent commit